src/schnetpack/transform/neighborlist.py

```diff
@@ -31,6 +33,23 @@ class CacheException(Exception):
     pass
 
 
+class AverageNumberOfNeighbors:
```

Comment:
This functionality is almost completely covered in [here](https://github.com/atomistic-machine-learning/schnetpack/blob/337fa3f803725201b5890ec4ee07da0d4774dec8/src/schnetpack/transform/neighborlist.py#L608
There is no additional benefit for now to include the average neighbors